public/js/stylehelpers/freelancer.js

```diff
@@ -72,7 +73,7 @@ var animateObjects = function() {
       $hero = $(".parallax-hero.red");
   
   $parallax.css({
-    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px "
+    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px ",
```

Comment:
Feel free to look at/review the linting settings. They're definitely causing goofiness in places.